docs/userguide/networking/get-started-overlay.md

```diff
@@ -0,0 +1,247 @@
+
+1. Log into a system prepared with the prerequisite Docker Engine, Docker Machine, and VirtualBox software.
+
+2. Provision a VirtualBox machine called `mh-keystore`.  
+
+				$ docker-machine create -d VirtualBox mh-keystore
+
+		When you provision a new machine, the process adds Docker Engine to the
+		host. This means rather than installing Consul manually, you can create an
+		instance using the [consul image from Docker
+		Hub](https://hub.docker.com/r/progrium/consul/). You'll do this in the next step.
+
+3. Start a `progrium/consul` container running on the `mh-keystore` machine.
+
+			docker $(docker-machine config mh-keystore) run -d \
+				-p "8500:8500" \
+				-h "consul" \
+				progrium/consul -server -bootstrap
+
+	 You passed the `docker run` command the connection configuration using a bash
+	 expansion `$(docker-machine config mh-keystore)`.  The client started a
+	 `progrium/consul` image running in the `mh-keystore` machine. The server is called `consul`and is listening port `8500`.
+
+4. Set your local environment to the `mh-keystore` machine.
+
+			$  eval "$(docker-machine env mh-keystore)"
+
+5. Run the `docker ps` command to see the `consul` container.
+
+			$ docker ps
+			CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                                                                            NAMES
+			4d51392253b3        progrium/consul     "/bin/start -server -"   25 minutes ago      Up 25 minutes       53/tcp, 53/udp, 8300-8302/tcp, 0.0.0.0:8500->8500/tcp, 8400/tcp, 8301-8302/udp   admiring_panini
+
+Keep your terminal open and move onto the next step.
+
+
+## Step 2: Create a Swarm cluster
+
+In this step, you use `docker-machine` to provision the hosts for your network. At this point, you won't actually created the network. You'll create several virtual hosts in VirtualBox. One of the host will act as th Swarm master, you'll create that first. As you create each host, you'll pass the Engine on that host options which are needed to create the `overlay` network.
+
+1. Create a swarm master.
+
+			docker-machine create \
+			-d VirtualBox \
+			--swarm --swarm-image="swarm" --swarm-master \
+			--swarm-discovery="consul://$(docker-machine ip mh-keystore):8500" \
+			--engine-opt="cluster-store=consul://$(docker-machine ip mh-keystore):8500"
+			--engine-opt="cluster-advertise=eth1:2376" \
+			mhs-demo0
+
+	At creation time, you supply the Engine `daemon` with the ` --cluster-store` option. This option tells the Engine the location of the key-value store for the `overlay` network. The bash expansion `$(docker-machine ip mh-keystore)` resolves to the IP address of the Consul server you created in "STEP 1". The `--cluster-advertise` option
+
+2. Create another host and add it to the Swarm cluster.
+
+		docker-machine create -d VirtualBox \
```

Comment:
Missing `$` prompt

Also, could you wrap `-d ..` to the next line, to be consistent with the previous example

(and `--drive`)